website/static/js/components/quickFiles.js

```diff
@@ -0,0 +1,150 @@
+var $osf = require('js/osfHelpers');
+var iconmap = require('js/iconmap');
+var lodashFind = require('lodash.find');
+var mHelpers = require('js/mithrilHelpers');
+var Raven = require('raven-js');
+
+var withPagination = require('js/components/pagination').withPagination;
+
+var QUICKFILES_PAGE_SIZE = 10;
+
+
+var _buildUrl = function(page, user) {
+
+    var query = {
+        'page[size]': QUICKFILES_PAGE_SIZE,
+        'page': page || 1,
+        'version': '2.2',
+    };
+
+    return $osf.apiV2Url('users/' + user +  '/quickfiles/', { query: query});
+};
+
+
+var _getNextItems = function(ctrl, url, updatePagination) {
+    if(ctrl.requestPending()) {
+        return;
+    }
+
+    ctrl.quickFiles([]);
+    ctrl.requestPending(true);
+
+    var promise = m.request({
+        method : 'GET',
+        url : url,
+        background : true,
+        config: mHelpers.apiV2Config({withCredentials: window.contextVars.isOnRootDomain})
+    });
+
+    promise.then(
+        function(result) {
+            ctrl.requestPending(false);
+            ctrl.quickFiles(result.data);
+            updatePagination(result, url);
+            m.redraw();
+            return promise;
+        }, function(xhr, textStatus, error) {
+            ctrl.failed = true;
+            ctrl.requestPending(false);
+            m.redraw();
+            Raven.captureMessage('Error retrieving quickfiles', {
+                extra: {
+                    url: url,
+                    textStatus: textStatus,
+                    error: error
+                }
+            });
+        }
+    );
+};
+
+
+var QuickFile = {
+
+    controller: function(options) {
+        var self = this;
+        self.file = options.file;
+        self.icon =  iconmap.file;
+    },
+
+    view: function(ctrl)  {
+        var viewBase = window.location.origin + '/quickfiles';
+        var viewUrl = ctrl.file.attributes.guid ? viewBase + '/' + ctrl.file.attributes.guid : viewBase + ctrl.file.attributes.path;
+        return m('div', [
+            m('li.project list-group-item list-group-item-node cite-container', [
+                m('h4.list-group-item-heading', [
+                    m('span.component-overflow.f-w-lg', {style: 'line-height: 1.5; width: 100%'}, [
+                        m('span.project-statuses-lg', {style: 'width: 50%; float:left'}, [
+                            m('span', {class: ctrl.icon, style: 'padding-right: 5px;'}, ''),
+                            m('a', {'href': viewUrl,
+                                onclick : function () {
+                                    $osf.trackClick('QuickFiles', 'view', 'view-quickfile-from-profile-page');
+                                }
+                            }, ctrl.file.attributes.name),
+                        ]),
+                        m('div', {style: 'width: 50%; float:right; font-size:small; line-height:2;'}, 'downloads: ' + ctrl.file.attributes.extra.downloads)
```

Comment:
This is fine, but for future reference, you can pass `style` as an object, e.g.

```javascript
{style: {width: '50%', ..., fontSize: 'small'}}
```

Reply:
Instead of making `font-size: small`, does it make sense to use a `<small>` tag?

Also, why is `line-height` being modified? There isn't a lot of text here, so it seems a little strange to change the spacing between lines. Would `margin` make sense?

Reply:
I hadn't really touched the meat of these specific styles and divs and whatnot as this was just moving existing things around, but I'll check on that!

Reply:
Oh ok, I didn't realize this code came from somewhere else. In that case, don't worry about this for now.

Reply:
Oh man I was so wrong about that -- it did not come from anywhere else! I'll fix it!